src/com/pentalog/pentastagiu/homework2/shop/ClientsOfShop.java

```diff
@@ -0,0 +1,42 @@
+package com.pentalog.pentastagiu.homework2.shop;
+//
+public class ClientsOfShop {
```

Comment:
tinand cont ca ai deja pachetul shop, nu e nevoie sa repeti asta in numele clasei, pachetul de va da informatia legata de tipul de client cu care lucram; totodata se foloseste denumirea la singular cand numim o clasa, deci ar fi fost suficient sa folosesti Client in pachetul shop